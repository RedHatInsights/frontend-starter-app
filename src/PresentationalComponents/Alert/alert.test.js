import Alert from './alert';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';

describe('Alert', () => {
    test('should render correctly', () => {
        const successAlert = mount(<Alert type="success">Some text</Alert>)
        const dangerAlert = mount(<Alert type="danger">Some text</Alert>)
        const warningAlert = mount(<Alert type="warning">Some text</Alert>)
        const infoAlert = mount(<Alert type="info">Some text</Alert>)
        expect(toJson(successAlert)).toMatchSnapshot();
        expect(toJson(dangerAlert)).toMatchSnapshot();
        expect(toJson(warningAlert)).toMatchSnapshot();
        expect(toJson(infoAlert)).toMatchSnapshot();
    });
});
