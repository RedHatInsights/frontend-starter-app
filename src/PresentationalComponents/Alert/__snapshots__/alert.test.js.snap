// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should render correctly 1`] = `
<Component
  type="success"
>
  <div
    className="pf-c-alert pf-is-success"
    role="alert"
  >
     
    Some text
     
  </div>
</Component>
`;

exports[`Alert should render correctly 2`] = `
<Component
  type="danger"
>
  <div
    className="pf-c-alert pf-is-danger"
    role="alert"
  >
     
    Some text
     
  </div>
</Component>
`;

exports[`Alert should render correctly 3`] = `
<Component
  type="warning"
>
  <div
    className="pf-c-alert pf-is-warning"
    role="alert"
  >
     
    Some text
     
  </div>
</Component>
`;

exports[`Alert should render correctly 4`] = `
<Component
  type="info"
>
  <div
    className="pf-c-alert pf-is-info"
    role="alert"
  >
     
    Some text
     
  </div>
</Component>
`;
